import/popups/ContextMenu.qml

```diff
@@ -0,0 +1,269 @@
+/****************************************************************************
+**
+** Copyright (C) 2013 Jolla Ltd.
+** Contact: Dmitry Rozhkov <dmitry.rozhkov@jollamobile.com>
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.WebView.Popups 1.0
+
+Rectangle {
+    id: root
+
+    property string linkHref
+    property string linkTitle
+    property string linkProtocol
+    property string imageSrc
+    property string contentType
+    property PageStack pageStack
+    property var tabModel // TODO: what about this one?
```

Comment:
Let's see... at least can be used now to hide open to in a new tab kind options.

When going forward we could move both historymodel and tabmodel under these components. Let's do not rush those now.